src/plugins/displayLayout/actions/CopyToClipboardAction.js

```diff
@@ -0,0 +1,34 @@
+import Clipboard from '@/utils/clipboard';
+
+export default class CopyToClipboardAction {
+    constructor(openmct) {
+        this.openmct = openmct;
+
+        this.cssClass = 'icon-duplicate';
+        this.description = 'Copy to Clipboard action';
+        this.group = "action";
+        this.key = 'copyToClipboard';
+        this.name = 'Copy to Clipboard';
+        this.priority = 9;
+    }
+
+    invoke(objectPath = null, viewContext) {
+        if (!objectPath) {
```

Comment:
why are we returning if we dont have an object path? we dont need an objectPath for this to work right?